src/main/java/model/Fruit.java

```diff
@@ -0,0 +1,66 @@
+package model;
+
+public class Fruit {
```

Comment:
let's rename this class as FruitTransaction as was suggested in README.md, because it contains not only name of the fruit but transaction details - operation and quantity.